aether-ui/aether/ui/api/utils.py

```diff
@@ -442,15 +376,9 @@ def linked_pipeline_object(object_name, id):
 
 
 def kernel_to_pipeline():
-    mappingsets_response = kernel_data_request('mappingsets/')
+    url = f'{utils.get_kernel_server_url()}/mappingsets/'
+    mappingsets = utils.get_all_docs(url)
     pipelines = []
-    while True:
-        mappingsets = mappingsets_response['results']
-        for mappingset in mappingsets:
-            pipelines.append(create_new_pipeline_from_kernel(mappingset))
-        if mappingsets_response['next']:
-            url_segments = urlparse(mappingsets_response['next'])
-            mappingsets_response = kernel_data_request('{}?{}'.format(url_segments.path, url_segments.query)[1:])
-        else:
-            break
+    for mappingset in mappingsets:
```

Comment:
If you are not using the returned pipelines, you can save memory and resources not creating the array.

Reply:
Any action here?